packages/react-tokens/package.json

```diff
@@ -29,8 +29,8 @@
     "clean": "rimraf dist"
   },
   "devDependencies": {
-    "@patternfly/patternfly": "6.3.0-prerelease.6",
-    "css": "^3.0.0",
+    "@adobe/css-tools": "^4.4.2",
```

Comment:
Context: the original css package was last updated in 2020 and we were getting build errors due to the `@starting-style` property from Core. This new package is a fork of the original but more up to date.